pex.go

```diff
@@ -138,14 +121,6 @@ func (me *pexMsgFactory) PexMsg() (ret pp.PexMsg) {
 	return
 }
 
-func mustNodeAddr(addr net.Addr) krpc.NodeAddr {
```

Comment:
Moving this is okay, but it might actually be relevant in non-test code and end up coming back.

Reply:
Please, no, the library has too much panics wired in already. See, if we skip an address which, somehow, ended up broken (haven't seen any by far), it's fine. No need to panic.